lib/generation/generators/pb_generator.dart

```diff
@@ -47,4 +48,23 @@ abstract class PBGenerator {
       )
     ''';
   }
+
+  void addTagToAnalytics(String tagName) {
```

Comment:
All analytics methods should be part of the same service/class IMO since they have the same responsibility